projects/packages/account-protection/src/class-config.php

```diff
@@ -11,9 +11,11 @@
  * Class Config
  */
 class Config {
-	public const TRANSIENT_PREFIX      = 'password_detection';
-	public const ERROR_CODE            = 'password_detection_validation_error';
-	public const ERROR_MESSAGE         = 'Password validation failed.';
-	public const EMAIL_SENT_EXPIRATION = 600; // 10 minutes
-	public const MAX_RESEND_ATTEMPTS   = 3;
+	public const PASSWORD_DETECTION_TRANSIENT_PREFIX      = 'password_detection';
+	public const PASSWORD_DETECTION_ERROR_CODE            = 'password_detection_validation_error';
+	public const PASSWORD_DETECTION_ERROR_MESSAGE         = 'Password validation failed.';
```

Comment:
Does this error message need translation support?

Reply:
ATM I don't believe its surfaced in a user-facing way anywhere, but regardless probably not a bad idea to do so anyway.

Reply:
Done in [5559bce](https://github.com/Automattic/jetpack/pull/41401/commits/5559bce0f9f75522bfecb924a3cf53dcee87cc68)